packages/contracts/src/dollar/core/StakingShare.sol

```diff
@@ -29,24 +35,26 @@ contract StakingShare is ERC1155Ubiquity {
     uint256 private _totalLP;
     uint256 private _totalSupply;
 
+    IAccessControl public accessCtrl;
+
     // ----------- Modifiers -----------
-    modifier onlyMinter() override {
+    modifier onlyMinter() {
         require(
             accessCtrl.hasRole(STAKING_SHARE_MINTER_ROLE, msg.sender),
             "Staking Share: not minter"
         );
         _;
     }
 
-    modifier onlyBurner() override {
+    modifier onlyBurner() {
```

Comment:
Inherit from the `ERC1155Ubiquity` contract and keep the `override`

Reply:
done